source/pages/Transactions/TransactionConfirmation.tsx

```diff
@@ -106,7 +106,7 @@ const TransactionConfirmation: React.FC<ITransactionConfirmation> = ({
       setLoading(false);
       setSubmitted(true);
 
-      dispatchBackgroundEvent(`tx${type}.${host}`, response);
+      dispatchBackgroundEvent('tx', { ...response, type: `tx${type}.${host}` });
```

Comment:
To test this: The dapp receives a response after a tx (or request). It will be 'fail' if the event is not dispatched or the popup closes before finish

Reply:
yeah forgot to remove this, i was testing the background events and autolock